Packs/Illumio/Integrations/IllumioCore/IllumioCore.yml

```diff
@@ -0,0 +1,1659 @@
+      type: String
+    - contextPath: Illumio.TrafficFlows.dst.workload.hostname
+      description: Destination workload hostname.
+      type: String
+    - contextPath: Illumio.TrafficFlows.dst.workload.name
+      description: Destination workload name.
+      type: String
+    - contextPath: Illumio.TrafficFlows.dst.workload.href
+      description: Destination workload URI.
+      type: String
+    - contextPath: Illumio.TrafficFlows.dst.workload.os_type
+      description: Destination workload OS type.
+      type: String
+    - contextPath: Illumio.TrafficFlows.dst.workload.labels.href
+      description: Destination label URI.
+      type: String
+    - contextPath: Illumio.TrafficFlows.dst.workload.labels.key
+      description: Destination workload label key.
+      type: String
+    - contextPath: Illumio.TrafficFlows.dst.workload.labels.value
+      description: Destination workload label value.
+      type: String
+    - contextPath: Illumio.TrafficFlows.service.port
+      description: Port of the traffic.
+      type: Number
+    - contextPath: Illumio.TrafficFlows.service.proto
+      description: Protocol number of the traffic.
+      type: Number
+    - contextPath: Illumio.TrafficFlows.num_connections
+      description: Number of traffic flows reported in connections.
+      type: Number
+    - contextPath: Illumio.TrafficFlows.policy_decisions
+      description: Indicates the policy decision for the flow. Indicates if the traffic
+        flow is allowed, potentially blocked (but allowed), or blocked.
+      type: String
+    - contextPath: Illumio.TrafficFlows.state
+      description: State of the flow.
+      type: String
+    - contextPath: Illumio.TrafficFlows.flow_direction
+      description: Flow direction of the traffic.
+      type: String
+    - contextPath: Illumio.TrafficFlows.dst_bi
+      description: Bytes received till now by the destination over the flow during
+        the interval.
+      type: Number
+    - contextPath: Illumio.TrafficFlows.dst_bo
+      description: Bytes sent till now by the destination over the flow during the
+        interval.
+      type: Number
+    - contextPath: Illumio.TrafficFlows.timestamp_range.last_detected
+      description: Time range when traffic was last detected.
+      type: Date
+    - contextPath: Illumio.TrafficFlows.timestamp_range.first_detected
+      description: Time range when traffic was first detected.
+      type: Date
+  - arguments:
+    - default: false
+      description: Virtual service name.
+      isArray: false
+      name: name
+      required: true
+      secret: false
+    - default: false
+      description: Port number.
+      isArray: false
+      name: port
+      required: true
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      defaultValue: TCP
+      description: Communication protocol.
+      isArray: false
+      name: protocol
+      predefined:
+      - TCP
+      - UDP
```

Comment:
you are using upper case in a part of the time a lower in other.
consider to you a single standard here.